src/app/bungie-api/authenticated-fetch.ts

```diff
@@ -99,7 +98,11 @@ export async function getActiveToken(): Promise<Tokens> {
   try {
     return await getAccessTokenFromRefreshToken(token.refreshToken!);
   } catch (e) {
-    return await handleRefreshTokenError(e);
+    if (e instanceof Error || e instanceof Response) {
+      return await handleRefreshTokenError(e);
+    } else {
+      throw e;
+    }
```

Comment:
In the case of `e instanceof Error`, `handleRefreshTokenError` rethrows too, which is slightly confusing control flow -- maybe `handleRefreshTokenError` could also just take `unknown` and handle all rethrowing?